explainer.md

```diff
@@ -137,10 +137,10 @@ Same-origin iframes also create a new global object which is synchronously acces
 Sites like Salesforce.com make extensive use of same-origin iframes to create such global objects. Our experience with same-origin iframes motivated us to steer this proposal forward, which has the following advantages:
 
 - Frameworks would be able to better craft the available API within the global object of the ShadowRealm, aiming for what is necessary to evaluate the program.
-- Tailoring up [the exposed set of APIs into the code](#VirtualizedEnvironment) within the ShadowRealm provides a better developer experience for a less expensive work compared to tailoring down a full set of exposed APIs - e.g. iframes - that includes handling presence of `[LegacyUnforgeable]` attributes like `window.top`.
+- Tailoring up [the exposed set of APIs into the code](#VirtualizedEnvironment) within the ShadowRealm provides a better developer experience for a less expensive work compared to tailoring down a full set of exposed APIs — e.g. iframes — that includes handling presence of `[LegacyUnforgeable]` attributes like `window.top`.
```

Comment:
I'd rather use the common ASCII dash/hyphen character - U+002D in this explainer's source.

Your change seems to include the "em dash" instead (U+2014). 

Would you mind remove these parts? Thanks!

Reply:
Sure, done!

Reply:
Thank you!